src/ModAttachr.ts

```diff
@@ -29,10 +31,15 @@ export class ModAttachr implements IModAttachr {
      * Transforms mod names to storage keys.
      */
     private readonly transformModName: ITransformModName;
-
+
+    /**
+     * Class to hold modkeys in order to improve readability.

```

Comment:
> `to improve readability`

Don't include "why"s unless they're absolutely necessary. This one is not.

A good strategy for these is to just rephrase the class comment.